server/tests/test_notebook_view.py

```diff
@@ -56,11 +57,11 @@ def test_notebook_view_old_revision(client, test_notebook):
     )
     assert resp.status_code == 200
     assert get_title_block(resp.content) == initial_revision.title
-    print(str(resp.content))
     assert get_script_block(resp.content, "iomd", "text/iomd") == initial_revision.content
     assert get_script_block_json(resp.content, "notebookInfo") == {
         "connectionMode": "SERVER",
         "files": [],
+        "fileSources": [],
```

Comment:
I think this might be referencing something from the notebook view which is no longer there?

In any case we should probably settle on using snake case for these properties, just to be consistent with what's there already (we could talk about modifying both these views and the API to use camel case for better integration on the client side, but that's probably best settled elsewhere).

Reply:
yes this should be removed actually. We don't ship `fileSources` to the notebook in the notebook view anymore.